packages/roosterjs-editor-plugins/lib/plugins/ImageResize/ImageResize.ts

```diff
@@ -328,7 +371,20 @@ export default class ImageResize implements EditorPlugin {
     }
 
     private isWest(direction: string): boolean {
-        return direction && direction.substr(1, 1) == 'w';
+        return direction && (direction.substr(0, 1) == 'w' || direction.substr(1, 1) == 'w');
+    }
+
+    private isSingleDirectionNS(direction: string): boolean {
+        return (
+            direction &&
+            (direction.substr(0, 1) == 'n' || direction.substr(0, 1) == 's') &&
```

Comment:
can't this just be written direction == "n" || direction == "s"?

private isSingleDirectionNS = (direction: string): boolean => direction == "n" || direction == "s";

Reply:
The problem with this approach is that it would work for the html generation (line 308) where the handle position is 'n' but this.direction (used in line 251, 262) is set to the element style cursor (line 192) which would be "n-resize". So the method of getting the substring works for either pos ('n') or cursor property ('n-resize'), but using == 'n' doesn't.

If we really wanted to make direction and pos consistent, we could change the HANDLE_POSITIONS array to "n-resize" format, but then we need to use substrings here anyway. Or we could make this.direction be only the 'n' or 'ne' substring from the 'n-resize' or 'ne-resize' style cursor but that is just as complex.

Another option would be to use regex here.

Reply:
I think in that case it may not be a good idea to reuse the CSS cursor value to check the direction.

How about add a new attribute to the DIV data-direction="..", then we always check this value instead of the CSS cursor. And the value will only contain "n", "w", ...